x/wasm/keeper/keeper.go

```diff
@@ -157,7 +157,7 @@ func (k Keeper) GetContractInfo(ctx sdk.Context, contractAddress sdk.AccAddress)
 	store := ctx.KVStore(k.storeKey)
 	contractBz := store.Get(types.GetContractInfoKey(contractAddress))
 	if contractBz == nil {
-		return types.ContractInfo{}, sdkerrors.Wrapf(types.ErrNotFound, "constractInfo %s", contractAddress.String())
+		return types.ContractInfo{}, sdkerrors.Wrapf(types.ErrNotFound, "contractInfo %s", contractAddress.String())
```

Comment:
Thanks for catching this :) 
, but we already fixed the in next release branch https://github.com/terra-money/core/blob/0c3ac0f2ffb865ba8e0ae51f6693285f1bbf060c/x/wasm/keeper/keeper.go#L160